src/components/info/info-panel.vue

```diff
@@ -19,16 +34,26 @@ const appStore = useAppStore()
     </template>
 
     <template v-slot:content>
-      <div class="text-white">
-        <ul class="list-disc pl-10">
-          <li>
-            {{ t(`A right click (tap and hold on mobile)...`, { ns: 'app' }) }}
-          </li>
-          <li>
-            {{ t(`A short click (tap on mobile)...`, { ns: 'app' }) }}
-          </li>
-        </ul>
-      </div>
+      <template v-if="map">
+        <div v-show="locationInfo" class="absolute">
```

Comment:
any reason why using v-if l.37 and v-show l.38? could this be done using only one if? (and use v-else l.43)

Reply:
v-show is needed so that the streetview component remains persistant, so that the dom reference in google code remains valid
v-if removes thecomponent from the dom

Reply:
Ok understood, I hope this won't break the v3 panel because I remember facing some issues when the v3 was just hidden.But maybe it won't be the case for your Streetview panel.